build/configs/rtl8730e/rtl8730e_download.sh

```diff
@@ -202,7 +182,8 @@ function post_download()
 	if test -f "${EXTERNAL}.bin"; then
 		[ -e ${EXTERNAL}.bin ] && rm ${EXTERNAL}.bin
 	fi
-	rm "${IMG_TOOL_PATH}/setting.txt"
+	rm "${IMG_TOOL_PATH}/USB_download_setting.txt"
+	rm "${IMG_TOOL_PATH}/USB_erase_setting.txt"
```

Comment:
If we download binaries through UART, those files are not existed. Then, no error about no file?

Reply:
Only remove these files if using usb download.